dao/src/integ-test/java/uk/gov/moj/sdt/dao/BulkSubmissionDaoTest.java

```diff
@@ -136,7 +137,8 @@ void testInsert() {
 
         LOGGER.debug("payload for bulk submission is {}", submission.getPayload());
```

Comment:
Does the output of getPayload() in this line also need to be converted to a String?